### PR TITLE
Add Spanish translation - Closes #774

### LINKED ISSUE #774: Add Spanish translation

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/localisation/es.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
+import Words from './words';
+
+const en: { [word in Words]: string } = {
```
Comment (ShawTim) [inline packages/tempus-client_v2/src/localisation/es.ts:3]: we may want to use `es` instead of `en` here, it works though.

**Review by ShawTim — APPROVED** [review]

**Review by ShawTim — COMMENTED** [review]
build fails due to missing key for `totalValueLocked`
reference:
https://github.com/tempus-finance/tempus-app/commit/82e1000400adffd8542a5392335319ff74b67a65

File: packages/tempus-client_v2/src/localisation/es.ts (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/localisation/es.ts:3]: good catch

**Comment by danieledamiani** [comment]
PR fixed and updated

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**